README.md

```diff
@@ -65,6 +75,11 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 Your Answer:
 ```js
 // write code here
+function doSomething (thingToDo) {
+  thingToDo;
```

Comment:
So close! You need to make sure to use parenthesis on this line to call the function passed in:
`thingToDo();`